lib/compute/oidc-config.ts

```diff
@@ -55,8 +52,8 @@ export class OidcConfig {
       'Job/Read',
     ];
 
-    public static addOidcConfigToJenkinsYaml(newYamlPath: string, admins?: string[]): void {
-      const jenkinsYaml: any = load(readFileSync(JenkinsMainNode.BASE_JENKINS_YAML_PATH, 'utf-8'));
+    public static addOidcConfigToJenkinsYaml(yamlObject: any, admins?: string[]): any {
```

Comment:
Would yamlObject be a Map?

Reply:
Nope it is an object https://github.com/opensearch-project/opensearch-ci/pull/90/files#diff-4c90022915859e4f2bab2c590f837062e3eb5a708c33a6d75d4ee20a149228f4R53

Reply:
Can it be a `Map` object?

Reply:
js-yaml load function returns `unknown` hence used `any` to perform operations on the same. 
We can enhance later is required.